### DISCUSSION TITLE
시험기간 대규모 메시지 트래픽 발생 시 Spring Boot 내부 메시지 브로커(STOMP)와 Kafka의 성능 비교: 메시지 처리량과 지연 시간 분석

### DISCUSSION THREAD

**Discussion by hhee3** [100-hours-a-week/discussions#47] [Deepdive]
스터디캠 특성상 학생들의 시험기간인 경우 유저가 급격히 증가한다.
현재 프로젝트에서 Spring Boot 내부 메시지 브로커를 사용하고 있는데, 대규모 트래픽이 몰릴 경우 어떻게 될 것인가?
1. 메시지 처리량 :  초당 처리 가능한 메시지수
2. 지연 시간: 메시지가 발행자에서 구독자로 전달되기까지 걸리는시간

**Comment by startupcodekr** [comment]
시험기간 대규모 메시지 트래픽 발생 시 Spring Boot 내부 메시지 브로커(STOMP)와 Kafka의 성능 비교: 메시지 처리량과 지연 시간 분석
